From 056f369564dc174149faa43a766d99e1767de264 Mon Sep 17 00:00:00 2001
From: Steve Kondik <steve@cyngn.com>
Date: Fri, 28 Oct 2016 13:38:12 -0700
Subject: [PATCH 062/249] cm: Remove AndroidAsync dependency

 * We still need to compile Ion ourselves due to patches and bugfixes
   which haven't been upstreamed, but we now declare our dependencies
   using Maven so this can be dropped.

Change-Id: Ib452e5d16c406e7aaebeb2cbe3239809a33ac9c1
---
 snippets/cm.xml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/snippets/cm.xml b/snippets/cm.xml
index 8628ca9..1775270 100644
--- a/snippets/cm.xml
+++ b/snippets/cm.xml
@@ -13,7 +13,6 @@
   <project path="external/gello-build" name="CyanogenMod/android_external_gello_build" />
   <project path="external/google" name="CyanogenMod/android_external_google" />
   <project path="external/htop" name="CyanogenMod/android_external_htop" />
-  <project path="external/koush/AndroidAsync" name="CyanogenMod/AndroidAsync" />
   <project path="external/koush/ion" name="CyanogenMod/ion" />
   <project path="external/libncurses" name="CyanogenMod/android_external_libncurses" />
   <project path="external/libphonenumbergoogle" name="CyanogenMod/android_external_libphonenumbergoogle" />
-- 
2.17.1

